docs/style-guide.md

```diff
@@ -5,7 +5,8 @@ There are really only 5 things you need to know:
 1. Use 4 spaces.
 2. Use [`stylish-haskell`](https://github.com/jaspervdj/stylish-haskell) for imports.
 3. Use spaces around all operators in all cases (exception: `(%)` from `Formatting`).
-4. Use `!` for all fields in `data` types.
-5. All imports should be either qualified or explicit, i.e. with enumeration of stuff that you're importing (exceptions: `Universum` and `Prelude`).
+4. As a default, use `!` to make all fields in `data` types strict. If strictness annotations are omitted, document why.
+5. As a default, make functions lazy. If they must be strict, document why. An exception to this rule is accumulators.
```

Comment:
Personally I wouldn't give an exception for accumulators - they can be documented like everything else. But this is fine.

Reply:
I agree, since they needn't always be lazy. Will remove that bit.